automation_tools/__init__.py

```diff
@@ -172,7 +172,7 @@ def setup_ddns(entry_domain, host_ip):
     if ddns_hash is None:
         print('The DDNS_HASH environment variable should be defined')
         sys.exit(1)
-
+    execute(subscribe, host=host_ip)
```

Comment:
It will work, but hostname would be better instead of IP.

Reply:
I tried that earlier and failed with `AttributeError: 'NoneType' object has no attribute 'strip'`. please take a look 185 job output